robottelo/ui/container.py

```diff
@@ -144,7 +144,7 @@ def delete(self, resource_name, container_name, really=True):
     def set_power_status(self, resource_name, cont_name, power_on):
         """Perform power on or power off for container
 
-        :param boolean power_on: True - for On, False - for Off
+        :param bool power_on: True - for On, False - for Off
 
```

Comment:
@sghai, Fixed this to reduce the warnings related to the `make docs-clean docs`.